vpr/src/pack/cluster_util.h

```diff
@@ -278,11 +280,16 @@ enum e_block_pack_status try_place_atom_block_rec(const t_pb_graph_node* pb_grap
                                                   int verbosity,
                                                   const int feasible_block_array_size);
 
-enum e_block_pack_status atom_cluster_floorplanning_check(const AtomBlockId blk_id,
-                                                          const ClusterBlockId clb_index,
-                                                          const int verbosity,
-                                                          PartitionRegion& temp_cluster_pr,
-                                                          bool& cluster_pr_needs_update);
+bool atom_cluster_floorplanning_check(AtomBlockId blk_id,
```

Comment:
I know this one isn't yours, but it would be good to add a Doxygen comment if you know its functionality.